odbc_loader_package/Tests.out

```diff
@@ -7,28 +8,28 @@ CREATE TABLE
  i | b |  f  |   v    |   c    |   lv   |       bn       |   vb   |  lvb   |     d      |    t     |         ts          |     tz      |          tsz           |      n      
 ---+---+-----+--------+--------+--------+----------------+--------+--------+------------+----------+---------------------+-------------+------------------------+-------------
    |   |     |        |        |        |                |        |        |            |          |                     |             |                        |            
- 1 | t | 1.5 | test 1 | test 1 | test 1 | test 1\000\000 | test 1 | test 1 | 2000-01-01 | 00:01:04 | 2038-01-01 08:14:07 | 01:21:00-04 | 2000-06-01 03:21:00-04 | 123456.7890
- 2 | t | 2.5 | test 2 | test 2 | test 2 | test 2\000\000 | test 2 | test 2 | 2000-01-02 | 00:02:04 | 2038-01-02 08:14:07 | 01:22:00-04 | 2000-06-01 03:22:00-04 | 123456.7890
- 3 | t | 3.5 | test 3 | test 3 | test 3 | test 3\000\000 | test 3 | test 3 | 2000-01-03 | 00:03:04 | 2038-01-03 08:14:07 | 01:23:00-04 | 2000-06-01 03:23:00-04 | 123456.7890
- 4 | t | 4.5 | test 4 | test 4 | test 4 | test 4\000\000 | test 4 | test 4 | 2000-01-04 | 00:04:04 | 2038-01-04 08:14:07 | 01:24:00-04 | 2000-06-01 03:24:00-04 | 123456.7890
- 5 | t | 5.5 | test 5 | test 5 | test 5 | test 5\000\000 | test 5 | test 5 | 2000-01-05 | 00:05:04 | 2038-01-05 08:14:07 | 01:25:00-04 | 2000-06-01 03:25:00-04 | 123456.7890
- 6 | t | 6.5 | test 6 | test 6 | test 6 | test 6\000\000 | test 6 | test 6 | 2000-01-06 | 00:06:04 | 2038-01-06 08:14:07 | 01:26:00-04 | 2000-06-01 03:26:00-04 | 123456.7890
- 7 | t | 7.5 | test 7 | test 7 | test 7 | test 7\000\000 | test 7 | test 7 | 2000-01-07 | 00:07:04 | 2038-01-07 08:14:07 | 01:27:00-04 | 2000-06-01 03:27:00-04 | 123456.7890
- 8 | t | 8.5 | test 8 | test 8 | test 8 | test 8\000\000 | test 8 | test 8 | 2000-01-08 | 00:08:04 | 2038-01-08 08:14:07 | 01:28:00-04 | 2000-06-01 03:28:00-04 | 123456.7890
- 9 | t | 9.5 | test 9 | test 9 | test 9 | test 9\000\000 | test 9 | test 9 | 2000-01-09 | 00:09:04 | 2038-01-09 08:14:07 | 01:29:00-04 | 2000-06-01 03:29:00-04 | 123456.7890
+ 1 | t | 1.5 | test 1 | test 1 | test 1 | test 1\000\000 | test 1 | test 1 | 2000-01-01 | 04:01:00 | 2038-01-01 03:14:07 | 01:21:00-05 | 2000-06-01 03:21:00-05 | 123456.7890
```

Comment:
Would it be appropriate to have some tests before 2000?

Remembering the "-1900" in some of the expressions above, are dates before 1900 valid?  Should there be tests for them?

For a good time, run "cal 09 1752"

Reply:
ok